src/screens/dialogs/upload-file-dialog.tsx

```diff
@@ -0,0 +1,82 @@
+import { useContext } from "react";
+import toast from "react-hot-toast";
+import { uploadfileService } from "../../services/files/upload-file.service";
+import { Dialog } from "../../components/Dialog";
+import {
+  FilesDialogsContext,
+  AuthContext,
+  AVAILABLE_DIALOGS,
+  FilesContext
+} from "../../context/index";
+
+interface INewFile {
+  isFile: boolean;
+  isReady: boolean;
+  name: string;
+  size: number;
+  uuid: string;
+}
+
+export const UploadFileDialog = () => {
+  // Dialog state
+  const { closeDialog, dialogsVisibilityState } =
+    useContext(FilesDialogsContext);
+  const isOpen = dialogsVisibilityState[AVAILABLE_DIALOGS.UPLOAD_FILE];
+  const { addFile, currentDirectory } = useContext(FilesContext);
+  const { session } = useContext(AuthContext);
+
+  const uploadFile = async () => {
+    const fileInput = document.querySelector<HTMLInputElement>("#myfile");
+    if (fileInput?.files?.length) {
+      const fileList = fileInput.files;
+      for (let i = 0; i < fileList.length; i++) {
```

Comment:
Since your input doesn't have the `multiple` attribute. It is not possible to upload multiple files and this `for` loop its not necessary. 

I suggest to add the `multiple` attribute to the input because it's required that the users are able to upload multiple files at once.